src/components/select/simple-select/simple-select.stories.mdx

```diff
@@ -0,0 +1,161 @@
+import { Meta, Props, Preview, Story } from '@storybook/addon-docs/blocks';
+import SimpleSelect from '.';
+import Option from '../option';
+
+<Meta title="Design System/Simple Select" />
+
+# Simple Select
+
+Select one of available options from the drop-down menu
+
+## Overview 
+
+Simple Select is a Carbon styled equivalent of HTML Select Element with optional Type To Search functionality.
+
+## Guidance, hints and tips 
+
+Always insert `Option` Components inside the `SimpleSelect`, analogous to the original `<select>` and `<option>` HTML Elements.
+
+To enable Type To Search functionality you have to add `hasTypeToSearch` prop.
+It will allow you to filter through the existing options leaving only these that matches the typed in text.
+
+## Quick Start 
+
+To use the SimpleSelect component you have to import two components: `<SimpleSelect />` and `<Option />`
+from `carbon-react/lib/components/select`.
+
+### Basic usage:
+
+<Preview>
+  <Story name="basic" parameters={{ info: { disable: true }}} >
+    <div style={{ height: '250px' }}>
+      <SimpleSelect name='simple' id='simple'>
+        <Option text='Amber' value='1' />
+        <Option text='Black' value='2' />
+        <Option text='Blue' value='3' />
+        <Option text='Brown' value='4' />
+        <Option text='Green' value='5' />
+        <Option text='Orange' value='6' />
+        <Option text='Pink' value='7' />
+        <Option text='Purple' value='8' />
+        <Option text='Red' value='9' />
+        <Option text='White' value='10' />
+        <Option text='Yellow' value='11' />
+      </SimpleSelect>
+    </div>
+  </Story>
+</Preview>
+
+### Opens on action:
```

Comment:
Please can we add more detail here.
I think the prop `opensOnFocus` should be `openOnFocus` and it should default to `false` just like the HTML component.

Reply:
I agree with the name change, but if that prop defaults to false then the select will be not backward compatible with the old one

Reply:
changed to `preventFocusAutoOpen` to keep backward compatibility

Reply:
This was kept as `openOnFocus` as it better describes the new default behaviour. `preventFocusAutoOpen` ends up being a double negative when you want it to open on focus `preventFocusAutoOpen=false`.